finch/processes/wps_geoseries_to_netcdf.py

```diff
@@ -0,0 +1,200 @@
+from pywps import ComplexInput, ComplexOutput, FORMATS, LiteralInput
+
+import logging
+from pathlib import Path
+from .wps_base import FinchProcess
+from . import wpsio
+from .utils import log_file_path, write_log, dataset_to_netcdf
+from xclim.core.formatting import update_history
+import xarray as xr
+import numpy as np
+import cf_xarray.geometry as cfgeo
+import geopandas as gpd
+from urllib.parse import urlparse
+
+LOGGER = logging.getLogger("PYWPS")
+
+
+class GeoseriesToNetcdfProcess(FinchProcess):
+    def __init__(self):
+        inputs = [
+            ComplexInput(
+                "resource",
+                "Geospatial series",
+                abstract="Series of geographical features.",
+                supported_formats=[FORMATS.GEOJSON],
+                min_occurs=1,
+                max_occurs=1,
+            ),
+            LiteralInput(
+                "index_dim",
+                "Index dimension",
+                abstract="Name of the column in the data to be converted into the index dimension.",
+                data_type="string",
+                min_occurs=1,
+                max_occurs=1,
+            ),
+            LiteralInput(
+                "feat_dim",
+                "Feature dimension",
+                abstract="Name of the column in the data to be converted into the 'feature' dimension. Each geometry must be different along and only along this dimension.",
+                data_type="string",
+                default="",
```

Comment:
Isn't that always "geometry" ?

Reply:
Usually. If the GeoSeries was built from a SQL request, the convention is to name that column "the_geom".

Reply:
That's not what I meant here, so I need to clarify the docs. 
When opening a geojson with geopandas and converting it to xarray, the variable containing the shapes is _always_ named "geometry" (regardless of what came in geopandas). After the reshape, the dimension is named "features".  And that dimension has a basic integer coordinate.

What the users are specifying here is the name of another column in the input table that could be used as the coordinate of the "features" dimension (and it also renames the dimension).

Reply:
Got it.